### ISSUE TITLE
Building OpenBLAS variant but not tagging correctly

### ISSUE THREAD

**Issue by jrmadsen** [conda-forge/scipy-feedstock#97]
Issue:

```
scipy                          1.2.1 py27_blas_openblash1522bff_0  conda-forge         
scipy                          1.2.1  py27h09a28d5_1  conda-forge         
scipy                          1.2.1  py27h7c811a0_0  pkgs/main           
scipy                          1.2.1  py27he2b7bc3_0  pkgs/main           
scipy                          1.2.1 py36_blas_openblash1522bff_0  conda-forge         
scipy                          1.2.1  py36h09a28d5_1  conda-forge         
scipy                          1.2.1  py36h7c811a0_0  pkgs/main           
scipy                          1.2.1  py36he2b7bc3_0  pkgs/main           
scipy                          1.2.1 py37_blas_openblash1522bff_0  conda-forge         
scipy                          1.2.1  py37h09a28d5_1  conda-forge         
scipy                          1.2.1  py37h7c811a0_0  pkgs/main           
scipy                          1.2.1  py37he2b7bc3_0  pkgs/main           
scipy                          1.3.0  py36hab63836_0  conda-forge         
scipy                          1.3.0  py37hab63836_0  conda-forge         
```

Note: no `blas_openblas` for 1.3.0 but 

```shell
$ ldd /opt/packages/python/anaconda/4.5/envs/tomopy-37-mkl/lib/python3.7/site-packages/scipy/linalg/_fblas.cpython-37m-x86_64-linux-gnu.so
	linux-vdso.so.1 (0x00007ffca77c7000)
	libopenblas.so.0 => /usr/local/lib/libopenblas.so.0 (0x00007ffaa495d000)
	libgfortran.so.4 => /opt/packages/python/anaconda/4.5/envs/tomopy-37-mkl/lib/python3.7/site-packages/scipy/linalg/../../../../libgfortran.so.4 (0x00007ffaa5a16000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007ffaa45bf000)
	libgcc_s.so.1 => /opt/packages/python/anaconda/4.5/envs/tomopy-37-mkl/lib/python3.7/site-packages/scipy/linalg/../../../../libgcc_s.so.1 (0x00007ffaa45aa000)
	libquadmath.so.0 => /opt/packages/python/anaconda/4.5/envs/tomopy-37-mkl/lib/python3.7/site-packages/scipy/linalg/../../../../libquadmath.so.0 (0x00007ffaa4576000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007ffaa4185000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007ffaa3f66000)
	libgfortran.so.3 => /usr/lib/x86_64-linux-gnu/libgfortran.so.3 (0x00007ffaa3c35000)
	/lib64/ld-linux-x86-64.so.2 (0x00007ffaa59ad000)
```
linked to OpenBLAS. Only works because I have `libopenblas.so` on my system but this is causing failures in Travis when I build a package that disables OpenBLAS due to MKL.

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /opt/packages/python/anaconda/4.5/envs/tomopy-37-mkl:
#
# Name                    Version                   Build  Channel
binutils_impl_linux-64    2.31.1               h6176602_1  
binutils_linux-64         2.31.1               h6176602_7  
blas                      2.10                        mkl    conda-forge
bzip2                     1.0.6             h14c3975_1002    conda-forge
ca-certificates           2019.3.9             hecc5488_0    conda-forge
cairo                     1.16.0            ha4e643d_1000    conda-forge
certifi                   2019.3.9                 py37_0    conda-forge
cftime                    1.0.3.4         py37hd352d35_1001    conda-forge
cloudpickle               1.0.0                      py_0    conda-forge
cmake                     3.14.4               hf94ab9c_0    conda-forge
coverage                  4.5.3            py37h14c3975_0    conda-forge
curl                      7.64.1               hf8cf82a_0    conda-forge
cycler                    0.10.0                     py_1    conda-forge
cytoolz                   0.9.0.1         py37h14c3975_1001    conda-forge
dask-core                 1.2.2                      py_0    conda-forge
dbus                      1.13.6               he372182_0    conda-forge
decorator                 4.4.0                      py_0    conda-forge
dxchange                  0.1.5                 py37_1000    conda-forge
dxfile                    0.5                        py_1    conda-forge
edffile                   5.0.0           pyh24bf2e0_1000    conda-forge
expat                     2.2.6                he6710b0_0  
ffmpeg                    4.1.3                h167e202_0    conda-forge
fontconfig                2.13.1            he4413a7_1000    conda-forge
freetype                  2.10.0               he983fc9_0    conda-forge
gcc_impl_linux-64         7.3.0                habb00fd_1    conda-forge
gcc_linux-64              7.3.0                h553295d_7    conda-forge
gettext                   0.19.8.1          hc5be6a0_1002    conda-forge
giflib                    5.1.9                h516909a_0    conda-forge
git                       2.21.0          pl526h2882143_0    conda-forge
glib                      2.58.3            hf63aee3_1001    conda-forge
gmp                       6.1.2             hf484d3e_1000    conda-forge
gnutls                    3.6.5             hd3a4fd2_1002    conda-forge
graphite2                 1.3.13            hf484d3e_1000    conda-forge
gst-plugins-base          1.14.4            hdf3bae2_1001    conda-forge
gstreamer                 1.14.4            h66beb1c_1001    conda-forge
gxx_impl_linux-64         7.3.0                hdf63c60_1    conda-forge
gxx_linux-64              7.3.0                h553295d_7    conda-forge
h5py                      2.9.0           nompi_py37hf008753_1102    conda-forge
harfbuzz                  2.4.0                h37c48d4_0    conda-forge
hdf4                      4.2.13            h9a582f1_1002    conda-forge
hdf5                      1.10.4          nompi_h3c11f04_1106    conda-forge
icu                       58.2              hf484d3e_1000    conda-forge
imageio                   2.5.0                    py37_0    conda-forge
intel-openmp              2019.3                      199  
jasper                    1.900.1           h07fcdf6_1006    conda-forge
jpeg                      9c                h14c3975_1001    conda-forge
kiwisolver                1.1.0            py37hc9558a2_0    conda-forge
krb5                      1.16.3            h05b26f9_1001    conda-forge
lame                      3.100             h14c3975_1001    conda-forge
libblas                   3.8.0                    10_mkl    conda-forge
libcblas                  3.8.0                    10_mkl    conda-forge
libcurl                   7.64.1               hda55be3_0    conda-forge
libedit                   3.1.20181209         hc058e9b_0  
libffi                    3.2.1             he1b5a44_1006    conda-forge
libgcc-ng                 8.2.0                hdf63c60_1  
libgfortran-ng            7.3.0                hdf63c60_0  
libiconv                  1.15              h516909a_1005    conda-forge
liblapack                 3.8.0                    10_mkl    conda-forge
liblapacke                3.8.0                    10_mkl    conda-forge
libnetcdf                 4.6.2             hbdf4f91_1001    conda-forge
libpng                    1.6.37               hed695b0_0    conda-forge
libssh2                   1.8.2                h22169c7_2    conda-forge
libstdcxx-ng              8.2.0                hdf63c60_1  
libtiff                   4.0.10            h648cc4a_1001    conda-forge
libuuid                   2.32.1            h14c3975_1000    conda-forge
libuv                     1.29.1               h516909a_0    conda-forge
libwebp                   1.0.2                h576950b_1    conda-forge
libxcb                    1.13              h14c3975_1002    conda-forge
libxml2                   2.9.9                h13577e0_0    conda-forge
matplotlib                3.1.0                    py37_0    conda-forge
matplotlib-base           3.1.0            py37h5f35d83_0    conda-forge
mkl                       2019.3                      199  
mkl-devel                 2019.3                      200  
mkl-include               2019.3                      199  
mkl_fft                   1.0.13           py37h516909a_1    conda-forge
ncurses                   6.1               hf484d3e_1002    conda-forge
netcdf4                   1.5.1.2          py37had58050_0    conda-forge
nettle                    3.4.1             h1bed415_1002    conda-forge
networkx                  2.3                        py_0    conda-forge
ninja                     1.9.0                h6bb024c_0    conda-forge
nose                      1.3.7                 py37_1002    conda-forge
numexpr                   2.6.9           py37h637b7d7_1000    conda-forge
numpy                     1.16.3           py37he5ce36f_0    conda-forge
olefile                   0.46                       py_0    conda-forge
opencv                    4.1.0            py37h4a2692f_3    conda-forge
openh264                  1.8.0             hdbcaa40_1000    conda-forge
openssl                   1.1.1b               h14c3975_1    conda-forge
packaging                 19.0                       py_0    conda-forge
pcre                      8.43                 he6710b0_0  
perl                      5.26.2            h516909a_1006    conda-forge
pillow                    6.0.0            py37he7afcd5_0    conda-forge
pip                       19.1.1                   py37_0  
pixman                    0.34.0            h14c3975_1003    conda-forge
pthread-stubs             0.4               h14c3975_1001    conda-forge
pyctest                   0.0.12           py37hfce76fc_1    conda-forge
pyparsing                 2.4.0                      py_0    conda-forge
pyqt                      5.9.2            py37hcca6a23_0    conda-forge
python                    3.7.3                h5b0a415_0    conda-forge
python-dateutil           2.8.0                      py_0    conda-forge
pywavelets                1.0.3            py37hd352d35_1    conda-forge
qt                        5.9.7                h52cfd70_1    conda-forge
readline                  7.0               hf8c457e_1001    conda-forge
rhash                     1.3.8                h1ba5d50_0  
scikit-build              0.9.0            py37hf484d3e_0    conda-forge
scikit-image              0.15.0           py37he1b5a44_0    conda-forge
scipy                     1.3.0            py37hab63836_0    conda-forge
setuptools                41.0.1                   py37_0    conda-forge
setuptools-scm-git-archive 1.0                      pypi_0    pypi
setuptools_scm            3.3.3                      py_0    conda-forge
setuptools_scm_git_archive 1.0                        py_1    conda-forge
sip                       4.19.8          py37hf484d3e_1000    conda-forge
six                       1.12.0                py37_1000    conda-forge
spefile                   1.6                        py_1    conda-forge
sqlite                    3.28.0               h7b6447c_0  
tifffile                  0.15.1          py37h3010b51_1001    conda-forge
timemory                  2.3.0                    py37_0    jrmadsen
tk                        8.6.9             h84994c4_1001    conda-forge
tomopy                    1.4.2.dev305+gd8bec21          pypi_0    pypi
toolz                     0.9.0                      py_1    conda-forge
tornado                   6.0.2            py37h516909a_0    conda-forge
wheel                     0.33.4                   py37_0    conda-forge
x264                      1!152.20180806       h14c3975_0    conda-forge
xorg-kbproto              1.0.7             h14c3975_1002    conda-forge
xorg-libice               1.0.9             h516909a_1004    conda-forge
xorg-libsm                1.2.3             h84519dc_1000    conda-forge
xorg-libx11               1.6.7             h14c3975_1000    conda-forge
xorg-libxau               1.0.9                h14c3975_0    conda-forge
xorg-libxdmcp             1.1.3                h516909a_0    conda-forge
xorg-libxext              1.3.4                h516909a_0    conda-forge
xorg-libxrender           0.9.10            h516909a_1002    conda-forge
xorg-renderproto          0.11.1            h14c3975_1002    conda-forge
xorg-xextproto            7.3.0             h14c3975_1002    conda-forge
xorg-xproto               7.0.31            h14c3975_1007    conda-forge
xz                        5.2.4             h14c3975_1001    conda-forge
zlib                      1.2.11            h14c3975_1004    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : tomopy-37-mkl
    active env location : /opt/packages/python/anaconda/4.5/envs/tomopy-37-mkl
            shell level : 2
       user config file : /home/jrmadsen/.condarc
 populated config files : /home/jrmadsen/.condarc
          conda version : 4.6.14
    conda-build version : 3.15.1
         python version : 3.7.2.final.0
       base environment : /opt/packages/python/anaconda/4.5  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/linux-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://conda.anaconda.org/jrmadsen/linux-64
                          https://conda.anaconda.org/jrmadsen/noarch
                          https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /opt/packages/python/anaconda/4.5/pkgs
                          /home/jrmadsen/.conda/pkgs
       envs directories : /opt/packages/python/anaconda/4.5/envs
                          /home/jrmadsen/.conda/envs
               platform : linux-64
             user-agent : conda/4.6.14 requests/2.19.1 CPython/3.7.2 Linux/4.15.0-48-generic ubuntu/18.04.2 glibc/2.27
                UID:GID : 1021:1023
             netrc file : None
           offline mode : False

```
</details>

**Comment by jakirkham** [comment]
@isuruf, do you have any thoughts here?

**Comment by isuruf** [comment]
I think these packages were marked broken.

**Comment by isuruf** [comment]
Fixed in #100
